fly/table_fly_ip.go

```diff
@@ -0,0 +1,117 @@
+package fly
+
+import (
+	"context"
+
+	"github.com/turbot/steampipe-plugin-fly/flyapi"
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+
+	provider "github.com/fly-apps/terraform-provider-fly/graphql"
+)
+
+//// TABLE DEFINITION
+
+func tableFlyIP(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "fly_ip",
```

Comment:
Just by looking at the APIs, I think the table should be renamed to `fly_ip_address`.

Reply:
Agree @misraved. Looking at the query endpoints and the console, the table name should be `fly_ip_address`. But, regarding the table naming, I was referring to the [terraform provider](https://registry.terraform.io/providers/fly-apps/fly/latest/docs/resources/ip), where it is using `fly_ip`.
I was a bit confused and then decided to go with terraform 😄 .

Reply:
Agreed, terraform nomenclature assumes priority 👍 .